packages/pigeon/lib/dart_generator.dart

```diff
@@ -970,7 +983,8 @@ if (${_varNamePrefix}replyList == null) {
   /// Converts Constructors from the pigeon AST to a `code_builder` Constructor
   /// for a ProxyApi.
   Iterable<cb.Constructor> _proxyApiConstructors(
-    Iterable<Constructor> constructors, {
+    Iterable<Constructor> constructors,
+    AstProxyApi api, {
```

Comment:
I don't think this variable is used in this method.

Reply:
oops meant to remove that